scripts/win32.bat

```diff
@@ -5,12 +5,27 @@ exit /b
 
 ----- Begin wsf script --->
 <job><script language="VBScript">
+
+Private OSDrive
+
 strComputer = "."
 
 Set objWMIService = GetObject("winmgmts:\\" & strComputer & "\root\cimv2")
 
 Set colDiskDrives = objWMIService.ExecQuery("SELECT * FROM Win32_DiskDrive")
 
+Set colOperatingSystems = objWMIService.ExecQuery ("SELECT SystemDrive FROM Win32_OperatingSystem")
+
+For Each objOperatingSystem in colOperatingSystems
+    On Error Resume Next
+    '' get the OS System Drive if exists
+    OSDrive = objOperatingSystem.Properties_("SystemDrive")   
+    If Err.Number <> 0 Then
+        OSDrive = Err.Description
```

Comment:
So if `objOperatingSystem.Properties_("SystemDrive")` fails, we assign `OSDrive` to `Err.Description`? Why is this? As I understand your intention was to set something that will always differ from `objLogicalDisk.DeviceID`, so `system: False` is printed when an error happens right? If so, maybe worth clarifying the intention in a comment, and potentially set `OSDrive` to something more obvious to the eye, like `false` (not sure if VBScript has the concept of `undefined`, `null`?)